src/Acuminator/Acuminator.Tests/Helpers/EmbeddedFileDataAttribute.cs

```diff
@@ -48,11 +51,20 @@ protected EmbeddedFileDataAttribute(string[] fileNames, string testFilePath)
 			{
 				if (String.IsNullOrWhiteSpace(fileName))
 					// ReSharper disable once LocalizableElement
-					throw new ArgumentException("File name cannot be empty", nameof (fileNames));
+					throw new ArgumentException("File name cannot be empty", nameof(fileNames));
+			}
+			if (!internalCodeFileNames.IsNullOrEmpty())
+			{
+				foreach (string internalCodeFileName in internalCodeFileNames)
+				{
+					if (String.IsNullOrWhiteSpace(internalCodeFileName))
+						throw new ArgumentException("File name cannot be empty", nameof(internalCodeFileName));
+				}
 			}
 
 			_prefix = GetPrefixFromTestFilePath(testFilePath);
-			_fileNames = fileNames;
+
+			_fileNames = internalCodeFileNames.IsNullOrEmpty() ? fileNames : fileNames.ToList().Concat(internalCodeFileNames.ToList()).ToArray();
```

Comment:
Why do you need `ToList()` here?
`fileNames.Concat(internalCodeFileNames).ToArray()`